tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>consists <strong>in</strong> adding &gt; consists <strong>of</strong> adding</li><li>visualisation &gt; visualization</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Done</p>